package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@okta/okta-signin-widget",
   "description": "The Okta Sign-In Widget",
-  "version": "4.2.0",
+  "version": "4.3.0",
```

Comment:
we dont change version, do we?

Reply:
How did that get in there? I'll clean this up